File: ProgrammingApplication-main/ProgrammingApplication-main/src/assignment_12/Person.java
package assignment_12;

public class Person{
	private String name;
	private String phoneNum;

	public Person(String n, String pn) {
		name = n;
		phoneNum = pn;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getPhoneNum() {
		return phoneNum;
	}

	public void setPhoneNum(String phoneNum) {
		this.phoneNum = phoneNum;
	}

}

File: C-Programming-main/C-Programming-main/20200284_김정현_발전기금관리/Person.cpp
#include "Person.h"
#include<iostream>
using namespace std;

Person::Person(string n, string phNum, int d)
{
	name = n;
	phoneNum = phNum;
	donation=d;
}

Person::Person()
{
	name = "";
	phoneNum = "";
	donation = 0;
}

Person::~Person()
{

}

string Person::getName() const
{
	return name;
}

void Person::setName(string n)
{
	name = n;
}

string Person::getPhoneNum() const
{
	return phoneNum;
}

void Person::setPhoneNum(string phNum)
{
	phoneNum = phNum;
}

int Person::getDonation() const
{
	return donation;
}

void Person::setDonation(int d)
{
	donation = d;
}

string Person::print() const
{
	return "Person";
}

string Person::getKey() const
{
	return string();
}

string Person::toString() const
{
	return string();
}
